composer.json

```diff
@@ -53,15 +57,22 @@
         "drupal-scaffold": "DrupalComposer\\DrupalScaffold\\Plugin::scaffold",
         "package": "Acquia\\Lightning\\Composer\\Package::execute",
         "release-version": "Acquia\\Lightning\\Composer\\ReleaseVersion::execute",
-        "verify-patched-constraints": "Acquia\\Lightning\\Composer\\PatchedConstraint::execute"
+        "verify-patched-constraints": "Acquia\\Lightning\\Composer\\PatchedConstraint::execute",
+        "enable-asset-packagist": "Acquia\\Lightning\\Composer\\AssetPackagist::execute"
     },
     "extra": {
+        "installer-types": [
+            "bower-asset",
+            "npm-asset"
+        ],
         "installer-paths": {
             "docroot/core": [
                 "type:drupal-core"
             ],
             "docroot/libraries/{$name}": [
-                "type:drupal-library"
+                "type:drupal-library",
+                "type:bower-asset",
+                "type:npm-asset"
```

Comment:
Technically yes, but in practice no, because we're providing a script to make the required modifications to composer.json. And that will only affect existing users of Lightning; it'll just work for new users.